src/features/defi/contexts/IdleProvider/idleInvestorSingleton.ts

```diff
@@ -8,7 +8,7 @@ import { getChainAdapterManager } from 'context/PluginProvider/chainAdapterSingl
 let _idleInvestor: IdleInvestor | undefined = undefined
 
 // we need to be able to access this outside react
-export const getIdleInvestor = (): IdleInvestor => {
+export const getIdleInvestor = (): IdleInvestor | undefined => {
```

Comment:
can revert this - this fn always returns `IdleInvestor`

Reply:
https://github.com/shapeshift/web/pull/3248/commits/2e26eac8cf3df5e9911eacf82d69f6b63cb59e4f